notebooks/tabular_ML_regression_icecream.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
I would suggest for this notebook to use all Icecream methods. For example you can also add ICE plots (or other plots).


Also, you can look at this notebook if you see other examples you can merge to your notebook (I'll send you the link)